plugin/src/main/java/com/denizenscript/denizen/events/item/InventoryPicksUpItemScriptEvent.java

```diff
@@ -49,7 +49,7 @@ public boolean couldMatch(ScriptPath path) {
         if (!path.eventArgLowerAt(1).equals("picks") || !path.eventArgLowerAt(2).equals("up")) {
             return false;
         }
-        if (path.eventArgLowerAt(0).equals("player")) {
+        if (couldMatchEntity(path.eventArgLowerAt(0))) {

```

Comment:
As discussed on discord: hopper minecarts are entities, so that needs some testing/thinking